src/Ordering.API/Apis/OrdersApi.cs

```diff
@@ -120,12 +120,14 @@ public static async Task<OrderDraftDTO> CreateOrderDraftAsync(CreateOrderDraftCo
         CreateOrderRequest request,
         [AsParameters] OrderServices services)
     {
+        
+        //mask the credit card number
+        
         services.Logger.LogInformation(
             "Sending command: {CommandName} - {IdProperty}: {CommandId} ({@Command})",
             request.GetGenericTypeName(),
             nameof(request.UserId),
-            request.UserId,
-            request);
+            request.UserId, new{}); //don't log the request as it has CC number
```

Comment:
Why not remove the `Command` parameter altogether?

Reply:
I'm not too sure what you're asking; this code is a bit hard for me to follow and I went with "least impact" wherever I could.

Reply:
I added a diff above to demonstrate